app/controllers/search_controller.rb

```diff
@@ -37,7 +37,7 @@ def search
   end
 
   def suggestions
-    search_term = Regexp.escape(params[:term].to_s)
+    search_term = Regexp.escape(params[:term].to_s.strip)
```

Comment:
Should we add a test for this to avoid regressions?

Reply:
yep, I agree